<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="HRP2JSKNT" >
  <xacro:include filename="HRP2JSKNT_body.urdf" />
  <xacro:include filename="HRP3HAND_L.urdf" />
  <xacro:include filename="HRP3HAND_R.urdf" />
</robot>
